docs/UserGuide.md

```diff
@@ -142,6 +142,20 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Deleting a meeting : `delete meeting`
+
+Deletes the specified meeting from the address book.
```

Comment:
I don't think the meetings list portion of our app is an "address book", so perhaps we should use a different term here?